enclave_wrangler/utils.py

```diff
@@ -211,51 +217,109 @@ def handle_paginated_request(first_page_url: str, verbose=False, called_by: str=
     return results
 
 
+def get_objects_df(object_type, outdir: str = None, save_csv=True, save_json=True):
```

Comment:
@Sigfried When refactoring `get_objects_by_type()` into `make_objects_request()`, I felt that this whole dataframe / saving results bit was not necessary. Only 2 functions were using `get_objects_by_type()` and made use of this, one of which was `download_favorite_objects()`.